test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,91 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
+
+	exit 1
+}
+
+
+while :; do
+	case "$1" in
+	-j)
+		parallel_jobs="$2"
+		[[ "$parallel_jobs" =~ ^[1-9][0-9]*$ ]] ||
+			fatal_msg "-j option argument must be a non-zero integer"
+		shift ; shift ;;
+	-*)
+		usage ;;
+	*)
+		break ;;
+	esac
+done
 
+# Wait for other format runs.
+wait_for_other_format_runs()
+{
+	while true
+	do
+		for process in ${PID_LIST[@]};do
+			if ps $process > /dev/null ; then
+				# The process id running so sleep for 5 second before checking another
+				# process status.
+				sleep 2
+			else
+				# Remove the process id of the completed runs from the process id array.
+				PID_LIST=(${PID_LIST[@]/$process})
+				exit_status=$?
```

Comment:
I'm not sure I understand how the exit_status is set. Without wait we don't get the actual status do we?

Reply:
Sorry, it was a miss. Fixed in 9a0edc7c0955bd9c66f73154014c2a14761f058e